contrib/lang/python/packages.el

```diff
@@ -172,7 +172,10 @@ which require an initialization must be listed explicitly in the list.")
                        'python-setup-shell))
     :config
     (progn
-      (add-hook 'inferior-python-mode-hook 'smartparens-mode)
+      (add-hook 'inferior-python-mode-hook '(lambda ()
+                                              (smartparens-mode)
+                                              (evil-search-highlight-persist -1)
+                                              (global-evil-search-highlight-persist -1)))
```

Comment:
Why globally disable it ?
I submitted a PR to remove the `:global t` in evil-search-highlight-persist. I hope it will be merged soon.